saleor/product/search.py

```diff
@@ -51,7 +58,28 @@ def prepare_product_search_document_value(
     return search_document.lower()
 
 
-def generate_product_fields_search_document_value(product: "Product"):
+def prepare_product_search_vector_value(
+    product: "Product", *, already_prefetched=False
+) -> SearchVector:
+    if not already_prefetched:
+        prefetch_related_objects([product], *PRODUCT_FIELDS_TO_PREFETCH)
+    search_vector = SearchVector(Value(product.name), weight="A") + SearchVector(
```

Comment:
Are you providing the search config? It's useful for languages other than English, as it would apply stemming, etc.
https://docs.djangoproject.com/en/4.0/ref/contrib/postgres/search/#changing-the-search-configuration

The downside is that AWS RDS supports only a limited number of languages and there's no option to upload a dictionary without filesystem access

Reply:
I explicitly don't provide a config so we don't have to deal with stop words and stemming for now.